pom.xml

```diff
@@ -220,6 +221,18 @@
                         </execution>
                     </executions>
                 </plugin>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-jar-plugin</artifactId>
+                    <version>3.3.0</version>
```

Comment:
Any particular reason for not using the latest 3.4.2 version ?

Reply:
Hi @lazee ,

Updated to 3.4.2 as requested